src/remotelibrarymanager.h

```diff
@@ -0,0 +1,32 @@
+#ifndef REMOTELIBRARYMANAGER_H
+#define REMOTELIBRARYMANAGER_H
+
+#include <QObject>
+#include <QtNetwork>
+#include <kiwix/library.h>
+#include <kiwix/manager.h>
+
+class RemoteLibraryManager : public QObject
+{
+    Q_OBJECT
+
+public:
+
+    RemoteLibraryManager();
+    ~RemoteLibraryManager() {}
+
+public:
+    void doUpdate(const QUrl &url);
+
+private:
+    QNetworkAccessManager m_networkManager;
+    QNetworkReply *mp_reply;
+
+signals:
+    void booksChanged(const QString&);
```

Comment:
This signal is emit when we get a response, not when the books have changed.
So it should be name "requestReceived", "newOpdsStream", ...